### PR TITLE
Initial commit from ESB-Connectors repo

### PR DESCRIPTION
**Pull request by elilmatha**

### CODE REVIEW

File: src/test/resources/testng.xml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    </listeners>
+
+    <test name="googledriveRest-Connector-Test" preserve-order="true" verbose="2">
+        <packages>
+            <package name="org.wso2.carbon.connector.integration.test.googledriveRest"/>
+        </packages>
+    </test>
+
+</suite>
+
```
Comment (kesavany) [inline src/test/resources/testng.xml:35]: Remove the empty lines.

File: org.wso2.carbon.connector.googledriverest.iml [hunk 2]
```diff
@@ -0,0 +1,220 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (Shakila) [inline org.wso2.carbon.connector.googledriverest.iml:1]: Please remove this module file created by IntelliJ IDEA.

File: pom.xml [hunk 3]
```diff
@@ -0,0 +1,317 @@
...
+            <groupId>org.wso2.carbon</groupId>
+            <artifactId>SecVerifier</artifactId>
+            <version>${carbon.kernel.version}</version>
+            <type>aar</type>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.wso2.esb</groupId>
+            <artifactId>org.wso2.carbon.esb.common</artifactId>
+            <version>4.8.0</version>
```
Comment (Shakila) [inline pom.xml:125]: Please move this version into properties.

File: src/main/resources/googledriverest-children/insertChild.xml [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+ ~  KIND, either express or implied.  See the License for the
+ ~  specific language governing permissions and limitations
+ ~  under the License.
+-->
+<template name="insertChild" xmlns="http://ws.apache.org/ns/synapse">
+    <parameter name="apiUrl" description="API Url to access the endpoint"/>
+    <parameter name="folderId" description="The ID of the folder"/>
+    <parameter name="childId" description="The ID of the child"/>
+    <parameter name="fields" description="The needed fields in the response"/>
+    <parameter name="childLink" description="The name of a resource type"/>
```
Comment (Shakila) [inline src/main/resources/googledriverest-children/insertChild.xml:24]: Seems the descriptions for childLink and selfLink are wrong.

File: src/main/resources/googledriverest-comments/createComment.xml [hunk 5]
```diff
@@ -0,0 +1,77 @@
...
+ ~  Unless required by applicable law or agreed to in writing,
+ ~  software distributed under the License is distributed on an
+ ~  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ ~  KIND, either express or implied.  See the License for the
+ ~  specific language governing permissions and limitations
+ ~  under the License.
+-->
+<template name="createComment" xmlns="http://ws.apache.org/ns/synapse">
+    <parameter name="apiUrl" description="API Url to access the endpoint"/>
+    <parameter name="fileId" description="The name of a resource type"/>
```
Comment (Shakila) [inline src/main/resources/googledriverest-comments/createComment.xml:21]: The descriptions for fileId and fields are wrong. Please check all parameter descriptions in this PR.

File: src/main/resources/googledriverest-revisions/updateRevision.xml [hunk 6]
```diff
@@ -0,0 +1,84 @@
...
+ ~  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ ~  KIND, either express or implied.  See the License for the
+ ~  specific language governing permissions and limitations
+ ~  under the License.
+-->
+<template name="updateRevision" xmlns="http://ws.apache.org/ns/synapse">
+    <parameter name="apiUrl" description="API Url to access the endpoint"/>
+    <parameter name="fileId" description="The id of the file"/>
+    <parameter name="revisionId" description="The id of the revision"/>
+    <parameter name="published" description="The name of a resource type"/>
```
Comment (Shakila) [inline src/main/resources/googledriverest-revisions/updateRevision.xml:23]: Description for published, fields, pinned, publishAuto and publishedOutsideDomain are wrong.

File: org.wso2.carbon.connector.googledriverest.iml (hunk 2 above)
Comment (elilmatha) [inline org.wso2.carbon.connector.googledriverest.iml:1]: Fixed in https://github.com/wso2-extensions/esb-connector-googledriverest/pull/5

File: pom.xml (hunk 3 above)
Comment (elilmatha) [inline pom.xml:125]: Fixed in https://github.com/wso2-extensions/esb-connector-googledriverest/pull/5

File: src/main/resources/googledriverest-children/insertChild.xml (hunk 4 above)
Comment (elilmatha) [inline src/main/resources/googledriverest-children/insertChild.xml:24]: Fixed in https://github.com/wso2-extensions/esb-connector-googledriverest/pull/5

File: src/main/resources/googledriverest-comments/createComment.xml (hunk 5 above)
Comment (elilmatha) [inline src/main/resources/googledriverest-comments/createComment.xml:21]: Fixed in https://github.com/wso2-extensions/esb-connector-googledriverest/pull/5

File: src/main/resources/googledriverest-revisions/updateRevision.xml (hunk 6 above)
Comment (elilmatha) [inline src/main/resources/googledriverest-revisions/updateRevision.xml:23]: Fixed in https://github.com/wso2-extensions/esb-connector-googledriverest/pull/5

File: src/test/resources/artifacts/AXIS2/config/test_axis2_server_9000.xml [hunk 7]
```diff
@@ -0,0 +1,378 @@
+<!--
```
Comment (hmrajas) [inline src/test/resources/artifacts/AXIS2/config/test_axis2_server_9000.xml:1]: Pleas add WSO2 Licence Header and remove other test_axis2_server files.

File: src/test/resources/testng.xml (hunk 1 above)
Comment (elilmatha) [inline src/test/resources/testng.xml:35]: Fixed in https://github.com/wso2-extensions/esb-connector-googledriverest/pull/5

File: src/test/resources/artifacts/AXIS2/config/test_axis2_server_9000.xml (hunk 7 above)
Comment (elilmatha) [inline src/test/resources/artifacts/AXIS2/config/test_axis2_server_9000.xml:1]: Fixed in https://github.com/wso2-extensions/esb-connector-googledriverest/pull/6/

**Comment by imesh** [comment]
Would you mind adding a description to the PR? 
Are we moving code from one repo to another via a PR?

**Pull request merged**